src/lightning/data/streaming/data_processor.py

```diff
@@ -798,7 +802,6 @@ def _create_process_workers(
             worker = DataWorkerProcess(
                 worker_idx,
                 self.num_workers,
-                begins[worker_idx],
```

Comment:
The "begins" is never used, or am I mistaken?

Reply:
It isn't required anymore. We can remove it.